static/scss/certificates/certificate.scss

```diff
@@ -0,0 +1,123 @@
+// sass-lint:disable mixins-before-declarations
+
+.certificate-wrapper {
+  padding: 20px 15px;
+  margin: 20px auto;
+  max-width: 1120px;
+}
+
+.certificate-holder {
+  margin: 0 auto;
+  max-width: 730px;
+}
+
+.certificate {
+  border: 2px solid #d9e0e3;
+  background: white;
+  padding: 35px 15px 0;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  text-align: center;
+  color: black;
+
+  .certificate-logo {
+    display: block;
+    max-width: 250px;
+    margin: 0 auto 15px;
+
+    img {
+      display: block;
+      width: 100%;
+    }
+  }
+
+  .institute-logo {
+    display: block;
+    max-width: 202px;
+    margin: 0 auto 30px;
+
+    img {
+      display: block;
+      width: 100%;
+    }
+  }
+
+  .certify-text {
+    display: block;
+    font-size: 18px;
+    line-height: 25px;
+    color: #6b6969;
+    font-style: italic;
+  }
+
+  .institute-text {
+    display: block;
+    font-size: 18px;
+    line-height: 25px;
+    color: #6b6969;
```

Comment:
Can we extract this color into a variable?

Reply:
I was thinking but @ahmed-arbisoft truly speaking the current `variables.scss` is really a confusing file. personally do not prefer to add there.

Reply:
No I meant just declare this variable at the top of this file, that would be fine as well.